tests/test_urlpatterns.py

```diff
@@ -62,7 +61,7 @@ def _test_trailing_slash(self, urlpatterns):
 
     def test_trailing_slash(self):
         urlpatterns = [
-            url(r'^test/$', dummy_view),
+            re_path(r'^test/$', dummy_view),
```

Comment:
Can we change these throughout from `re_path(r'^test/$', ...)` to `path('test/', ...)`?

Reply:
yes of course